Dockerfile

```diff
@@ -1,24 +1,32 @@
+# BUILD redisfab/redisgears-${OSNICK}:M.m.b-x64
+
+ARG OSNICK=buster
+
+#----------------------------------------------------------------------------------------------
+# FROM redisfab/redis-x64-${OSNICK}:5.0.5 AS builder
 FROM redis:latest AS builder
 
-RUN set -ex; \
-    apt-get update; \
-	apt-get install -y python git
+ADD . /build
+WORKDIR /build
 
-ADD . /redisgears
-WORKDIR /redisgears
+RUN ./deps/readies/bin/getpy2
+RUN python ./system-setup.py
```

Comment:
write a comment that we may need to remove the python command

Reply:
Removed "python".